openpype/hosts/tvpaint/lib.py

```diff
@@ -518,8 +441,8 @@ def fill_reference_frames(frame_references, filepaths_by_frame):
             continue
 
         # Get destination filepath
-        src_filepath = filepaths_by_frame[ref_idx]
-        dst_filepath = filepaths_by_frame[frame_idx]
+        src_filepath = frames_data_by_layer_id[ref_idx]["filepath"]
+        dst_filepath = frames_data_by_layer_id[frame_idx]["filepath"]
```

Comment:
Erreur lors d'un publish classique, sans custom_frame ni keep_frame_index:
\Users\hfarre\quad\OpenPype\openpype\hosts\tvpaint\plugins\publish\extract_sequence.py", line 629, in _render_layer
    fill_reference_frames(frame_references, filepaths_by_frame)
  File "C:\Users\hfarre\quad\OpenPype\openpype\hosts\tvpaint\lib.py", line 444, in fill_reference_frames
    src_filepath = frames_data_by_layer_id[ref_idx]["filepath"]
TypeError: string indices must be integers

Reply:
Fixed